components/webui/client/src/pages/SearchPage/SearchControls/Presto/SqlQueryInput/index.tsx

```diff
@@ -0,0 +1,28 @@
+import {useCallback} from "react";
+
+import SqlEditor from "../../../../../components/SqlEditor";
+import useSearchStore from "../../../SearchState/index";
+import styles from "./index.module.css";
+
+
+/**
+ * Renders SQL query input.
+ *
+ * @return
+ */
+const SqlQueryInput = () => {
+    const handleChange = useCallback((value: string | undefined) => {
+        const {updateQueryString} = useSearchStore.getState();
+        updateQueryString(value || "");
+    }, []);
+
+    return (
+        <div className={styles["input"] || ""}>
+            <SqlEditor
+                height={"150px"}
```

Comment:
i find it kinda awkward to show a giant box if the user hasn't input any query / the query is just a single line. Let's see if we can get the editor to auto resize base on the content
```suggestion
```